docs/en/latest/profile.md

```diff
@@ -57,15 +57,27 @@ export APISIX_NODE_LISTEN=8132
 export DEPLOYMENT_ADMIN_ADMIN_LISTEN=9232
 ```
 
```

Comment:
I know it sounds stupid, but please sum up like: at this point, Apisix listens on port 8132 and ...